src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/SnakeRoadCheck.java

```diff
@@ -97,10 +97,10 @@ public boolean validCheckForObject(final AtlasObject object)
         this.markAsFlagged(object.getOsmIdentifier());
 
         // Instantiate the network walk with the starting edge
-        SnakeRoadNetworkWalk walk = initializeNetworkWalk(edge);
+        final SnakeRoadNetworkWalk walk = initializeNetworkWalk(edge);
 
         // Walk the road
-        walk = walkNetwork(edge, walk);
+        walkNetwork(edge, walk);
```

Comment:
It looks like we don't do anything with the return value - maybe the function can be void

Reply:
I will modify it to void.